pkgs/development/tools/jira-cli-go/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, buildGoModule, fetchFromGitHub, less, more, installShellFiles }:
+
+buildGoModule rec {
+  pname = "jira-cli-go";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "ankitpokhrel";
+    repo = "jira-cli";
+    sha256 = "sha256-sPoFv3Gzue5H6TJuQZJvqB/Dx/URp9Kt2UuIvKSnAxg=";
+  };
+
+  vendorSha256 = "sha256-UO30/D65vpu3PgEsfSDL3nYgkwo5Cj+1WKiokk7KKKg=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/ankitpokhrel/jira-cli/internal/version.GitCommit=${src.rev}"
+    "-X github.com/ankitpokhrel/jira-cli/internal/version.SourceDateEpoch=0"
+    "-X github.com/ankitpokhrel/jira-cli/internal/version.Version=${version}"
+  ];
+
+  checkInputs = [ less more ]; # Tests expect a pager in $PATH
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/jira version | grep ${version} > /dev/null
+  '';
```

Comment:
You can use the `testVersion` instead.

Reply:
Nice to know it :P will need to refactor my packages.